pkg/compact/compact.go

```diff
@@ -31,6 +31,8 @@ const (
 	ResolutionLevelRaw = ResolutionLevel(downsample.ResLevel0)
 	ResolutionLevel5m  = ResolutionLevel(downsample.ResLevel1)
 	ResolutionLevel1h  = ResolutionLevel(downsample.ResLevel2)
+
+	consistencyDelay = time.Duration(30 * time.Minute)
```

Comment:
so I feel we have `syncTime`. I like `consistencyTime` name better, so can we just reuse and rename variable?